libr/debug/p/native/linux/linux_debug.c

```diff
@@ -517,3 +521,1439 @@ RList *linux_desc_list (int pid) {
 	closedir (dd);
 	return ret;
 }
+
+
+/* Coredump  functions */
+
+static map_file_t mapping_file = {0,0};
+
+static char *prpsinfo_get_fname(char *buffer) {
+	char *p;
```

Comment:
Wat. This function can be removed or at least inlined and changed by return strdup. The local var and assignment are unnecessary